src/helpers/redirectToCheckout.ts

```diff
@@ -20,7 +20,11 @@ const redirectToCheckout = (
     await stripe.redirectToCheckout({
       lineItems: [
         {
-          price: `${process.env.REACT_APP_STRIPE_PRICE_ID}`,
+          price: `${process.env.REACT_APP_STRIPE_PRICE_ID_1}`,
+          quantity: 1,
+        },
+        {
+          price: `${process.env.REACT_APP_STRIPE_PRICE_ID_2}`,
```

Comment:
these should be called:
`REACT_APP_STRIPE_MAGAZINE_PRICE_ID`
`REACT_APP_STRIPE_SHIPPING_PRICE_ID`

Reply:
yeeeah